src/basic/PlutoUI.jl.jl

```diff
@@ -359,8 +352,19 @@ DownloadButton(poem, "poem.txt")
 # ╔═╡ 067cbcde-cc4c-11ea-3eed-972dc6d7bb3b
 DownloadButton([0x01, 0x02, 0x03], "secret_data.bin")
 
-# ╔═╡ f69a5d5e-c765-11ea-3fa0-230c6c619730
-space
+# ╔═╡ 909057da-6f9d-443e-b58f-6c75ee21ab63
+
+
+# ╔═╡ 51a717b3-0658-4ead-889a-f89bf541e579
+md"## details"
+
+# ╔═╡ ef4c3dc2-3cfc-4888-87eb-fd495f785e69
+md"""
+As another output element, `PlutoUI` provides a `details` function to create a block with an always-visible title and foldable further content. This is useful for showing overly verbose details that would disrupt the normal flow of the text, or to hide, e.g., solutions of exercises in a tutorial notebook.
```

Comment:
Thanks for the reminder! How's this?

I'm not particular to any specific text. The main point is really just for `details` to be mentioned somewhere in the notebook, so that it's not an undocumented feature. So I'll be happy to tweak this any way you like